packages/govuk-frontend/src/govuk/components/summary-list/summary-list.yaml

```diff
@@ -8,63 +8,78 @@ params:
         type: string
         required: false
         description: Classes to add to the row `div`.
-      - name: key.text
-        type: string
-        required: true
-        description: If `html` is set, this is not required. Text to use within each key. If `html` is provided, the `text` option will be ignored.
-      - name: key.html
-        type: string
-        required: true
-        description: If `text` is set, this is not required. HTML to use within each key. If `html` is provided, the `text` option will be ignored.
-      - name: key.classes
-        type: string
-        required: false
-        description: Classes to add to the key wrapper.
-      - name: value.text
-        type: string
-        required: true
-        description: If `html` is set, this is not required. Text to use within each value. If `html` is provided, the `text` option will be ignored.
-      - name: value.html
-        type: string
+      - name: key
+        type: object
         required: true
-        description: If `text` is set, this is not required. HTML to use within each value. If `html` is provided, the `text` option will be ignored.
-      - name: value.classes
-        type: string
-        required: false
-        description: Classes to add to the value wrapper.
-      - name: actions.classes
-        type: string
-        required: false
-        description: Classes to add to the actions wrapper.
-      - name: actions.items
-        type: array
-        required: false
-        description: Array of action item objects.
+        description: The key of each row.
         params:
-          - name: href
+          - name: text
+            type: string
+            required: true
+            description: If `html` is set, this is not required. Text to use within each key. If `html` is provided, the `text` option will be ignored.
+          - name: html
             type: string
             required: true
-            description: The value of the link's `href` attribute for an action item.
+            description: If `text` is set, this is not required. HTML to use within each key. If `html` is provided, the `text` option will be ignored.
+          - name: classes
+            type: string
+            required: false
+            description: Classes to add to the key wrapper.
+      - name: value
+        type: object
+        required: true
+        description: The value of each row.
+        params:
           - name: text
             type: string
             required: true
-            description: If `html` is set, this is not required. Text to use within each action item. If `html` is provided, the `text` option will be ignored.
+            description: If `html` is set, this is not required. Text to use within each value. If `html` is provided, the `text` option will be ignored.
```

Comment:
Same thing here too!

Reply:
😭 Added another Trello card: https://trello.com/c/bYRTm149/21-for-options-in-arrays-of-objects-sometimes-we-say-eg-title-for-this-section-sometimes-title-for-each-section-sometimes-title-for